meinberlin/apps/moderationtasks/filters.py

```diff
@@ -1,25 +1,21 @@
-from rest_framework.filters import BaseFilterBackend
+from django_filters import rest_framework as rest_filters
 
 from meinberlin.apps.moderationtasks.models import ModerationTask
 
 
-class ModerationTaskFilterBackend(BaseFilterBackend):
-    """Filter out proposals that have the moderation tasks completed."""
-
-    def filter_queryset(self, request, queryset, view):
-
-        if "open_task" in request.GET:
-            task_id = request.GET["open_task"]
-            try:
-                moderation_task = ModerationTask.objects.get(id=task_id)
-                proposals_completed = getattr(
-                    moderation_task,
-                    "{app_label}_{model}_completed".format(
-                        app_label=queryset.model._meta.app_label,
-                        model=queryset.model.__name__.lower(),
-                    ),
-                ).all()
-                return queryset.exclude(id__in=proposals_completed)
-            except ModerationTask.DoesNotExist:
-                pass
+class OpenTaskFilter(rest_filters.ModelChoiceFilter):
+    def filter(self, queryset, value):
```

Comment:
I am a bit sad that the whole doc string is gone. Maybe we can change it to sth like "Filter out propsals with the field associated to it. Currently used to filter out proposals that have the moderation tasks completed."